src/wallet/rpcwallet.cpp

```diff
@@ -769,24 +499,23 @@ UniValue getbalance(const UniValue& params, bool fHelp)
 
     LOCK2(cs_main, pwalletMain->cs_wallet);
 
-    if (params.size() == 0)
-        return  ValueFromAmount(pwalletMain->GetBalance());
+    const UniValue& dummy_value = params[0];
+    if (!dummy_value.isNull() && dummy_value.get_str() != "*") {
```

Comment:
This makes sense contextually from upstream, but given that we've always told people in Zcash to hard-code this to the empty string, that should be our default.